### ISSUE TITLE
General Settings: Default Branch/Build Selection broken?

### ISSUE THREAD

**Issue by bruderol** [scenarioo/scenarioo#486]
it seems I can not properly configure default branch to select on scenarioo start in current RC for 2.2

When I sleect a branch it gets selected or sometimes not. 
Only after save I see the builds that can be selected for that branch.
And after save, when browsing a little bit around and reloading the settings page, it seems that my configured "default branch" did not get saved correctly.

@adiherzog Not sure if this is a bug??

**Comment by adiherzog** [comment]
@bruderol The behaviour "Only after save I see the builds that can be selected for that branch." is very old. This is something we could improve in our camp, but I don't have time to fix this now before release 2.2.

**Comment by bruderol** [comment]
yes, I know about this known issue, but it seemed to me that it is even somehow worse now.

**Comment by bruderol** [comment]
@adiherzog please do not worry too much about issues that I might open know. I know that you can not fix all of them. We can decide on Monday together shortly what is important of the leftovers and whether I could support you with fixing some, okay?

**Comment by adiherzog** [comment]
:+1:

**Comment by bruderol** [comment]
encountered today in my project, that the feature of default branch and build does not work (at least not in my project).

Then tested it with Adrian Herzog, did also not work, or at least seemed not to work.

but now when testing with release demo on my machine, it works :dizzy_face: 

hmm ...  not sure what to do about it ...

**Comment by bruderol** [comment]
I can not realy reproduce this problem. Even though I had this problem.
Retested many times with incognito browser (clean cookies).
And now it seems to work stable.

Probably there is another problem:
if the user once selected another build (or allready started scenarioo once), it will use the last selection from cookie. but if that last selection is not available, it should fallback again to default selection. Currently that is not the case and that might explain the strange seen behaviour in my current project.

**Comment by bruderol** [comment]
Okay, I give up, that is a different story, we cannot make this for release 3.0 happening anymore.

I opened a new issue #491
